__tests__/always-return.js

```diff
@@ -4,7 +4,7 @@ const rule = require('../rules/always-return')
 const RuleTester = require('eslint').RuleTester
 const ruleTester = new RuleTester({
   parserOptions: {
-    ecmaVersion: 6,
+    ecmaVersion: 11,
```

Comment:
Can we use `ES20XX` here instead?

Reply:
I think `ES20XX` makes it more clear as these are the ones that are more known/more used around the community.

Don't think @xjamundx would mind if we change this.

Reply:
Sounds like a plan!